plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -522,30 +522,45 @@ public function utilityController_exportProfiles_create($sender) {
 
         // Determine profile fields we need to add.
         $fields = $this->getProfileFields();
-        $columnNames = ['Name', 'Email', 'Joined', 'Last Seen', 'Discussions', 'Comments', 'Points', 'InviteUserID', 'InvitedByName'];
+        $columnNames = ['Name', 'Email', 'Joined', 'Last Seen', 'LastIpAddress', 'Discussions', 'Comments', 'Points', 'InviteUserID', 'InvitedByName', 'Location', 'Roles'];
```

Comment:
Can we keep the existing casing for `LastIPAddress`?

Reply:
Done!